source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -89,6 +89,25 @@ def label(self):
 
 class WordDocumentTextInfo(UIATextInfo):
 
+	def _get_locationText(self):
+		rects=self._rangeObj.getBoundingRectangles()
+		left=rects[0]

```

Comment:
This is going to fail in case of selections that are anchored at the start, in which case this will report the distances for the start, not for the cursor position.

I think you want to do something like this:

1. Copy the text info
2. If self.obj.isTextSelectionAnchoredAtStart, than collapse the copy to the end of the range. Collapse to the start otherwise
3. Now, you can safely get a bounding rectangle.

See also #8572, which implements a boundingRect property which we can use in the future.

Reply:
In the specific case this pr is for, locationText is only ever called on a collapsed textInfo at the reviewCursor position. Whether the reviewCursor is positioned at the correct end of the selection is somewhat a separate issue.

Reply:
Ah yes, agreed.